explorer/views/utils.py

```diff
@@ -2,6 +2,7 @@
 from django.db import DatabaseError
 
 from explorer import app_settings
+from explorer.chart import get_pie_chart, get_line_chart
```

Comment:
Lets rename the module as it already has 2 types of chart;

```suggestion
from explorer.charts import get_pie_chart, get_line_chart
```